app/lib/ds-routing/config/startCampaignTransitionsConfigModel.js

```diff
@@ -23,5 +23,5 @@ var startCampaignTransitionsConfigSchema = new mongoose.Schema({
 })
 
 module.exports = function(connection) {
-  return connection.model('start_campaign_transitions', startCampaignTransitionsConfigSchema);
+  return connection.model('start_campaign_transition', startCampaignTransitionsConfigSchema);
```

Comment:
was there a reason why you didn't change the string to `app.ConfigName.CAMPAIGN_TRANSITIONS` here?